NetSuiteProject/Exercise.js

```diff
@@ -2,22 +2,65 @@
 function add(x,y){
 	return x + y;
 };
+
+function addition(x,y){
+	if(typeof add (x,y) === "string"){
+	alert("This function only accepts Integers")
+	}
+	else {return x + y;}
+}
+
+
 //subtract
 function subtract(x, y){
 	return (x - y);
 };
+
+function subtraction(x, y){
+	if(isNaN(subtract(x,y))  === true){
+	alert("This function only accepts Integers");
+	}
+	else {return (x - y);}
+}
+
+
 //Multiplication
 function multiply(x, y){
 	return (x * y);
 };
+
+function multiplication(x,y){
+	if(isNaN(multiply(x,y)) === true ){
+		alert("This function only accepts Integers");
+	}
+	else {return (x * y);}
+}
+
+
 //Division
 function divide (x, y){
 	return (x / y);
 };
+
+function division(x, y){
+	if(isNaN(divide(x,y)) === true){
+		alert("This function only accepts Integers");
```

Comment:
I used the alert as the feedback Mechanism due to the fact that I can simply state the issue. Their is also a confirm message in which gives the specified message, along with an OK and a Cancel button. Their is also a prompt message allowing the user to input more information. This again will allow me to write more code further expanding the script.